docs/products/clickhouse/howto/clickhouse-query-cache.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use query cache in Aiven for ClickHouse®
+sidebar_label: Use query cache
+---
+
+Aiven for ClickHouse® provides a query cache mechanism that helps improve query performance
+by caching query results.
+
+## How it works
+
+When the Aiven for
+[ClickHouse query cache](https://clickhouse.com/docs/en/operations/query-cache) is enabled,
+multiple identical `SELECT` queries run simultaneously are computed only once,
+and further executions of the same query are served directly from the cache.
```

Comment:
How about this, break the sentence: 
```suggestion
. Subsequent executions of the same query are served directly from the cache.
```

..identical SELECT queries running simultaneously are computed only once. Subsequent executions of the same query are served directly from the cache.